---
layout: splash
classes: wide
author_profile: false
---

<body>
<h2>Peer-reviewed journal publications</h2>
<p style="margin-bottom: 0.5em;">*Corresponding author, †Co-first authors.</p>

<ol reversed align="justify">

<li>Y. Hirakata, <b>R. Mei</b>, K. Morinaga, T. Katayama, H. Tamaki, X. Meng, T. Watari, T. Yamaguchi, M. Hatamoto, and M.K. Nobu*, 2023. <b>Discovery of anaerobic bacterial scavengers of dead cells</b>. <i>The ISME Journal</i>. <a href="https://academic.oup.com/ismej/article/17/12/2279/7506654">10.1038/s41396-023-01538-2</a>.<br>
<p style="margin-top: 0.5em;">Cultivation and characterization of syntrophic <i>Bacteroidota</i> species that scavenge dead cells using distinct strategies and contribute to the cycle of life and death in oxygen-depleted environments.</p>

<li><b>R. Mei</b>, M. Kaneko, H. Imachi, and M.K. Nobu*, 2023. <b>The origin and evolution of methanogenesis and <i>Archaea</i> are intertwined</b>. <i>PNAS Nexus</i>. <a href="https://academic.oup.com/pnasnexus/article/2/2/pgad023/7010768">10.1093/pnasnexus/pgad023</a>.<br>
<p style="margin-top: 0.5em;">New evidence for the antiquity of methanogenesis and its influence on the transition from ancestral archaea to the diversity prominent today.</p>

<li><b>R. Mei</b> and W-T. Liu*, 2022. <b>Meta-omics supervised characterization of respiration activities associated with microbial immigrants in anaerobic sludge digesters</b>. <i>Environmental Science & Technology</i>. <a href="https://pubs.acs.org/doi/abs/10.1021/acs.est.2c01029">10.1021/acs.est.2c01029</a>.<br>
<p style="margin-top: 0.5em;">Meta-omics-supervised characterization of microbial immigrants in anaerobic sludge digesters and revealed their activities of anaerobic respiration.</p>

<li>J. Kim†, <b>R. Mei†</b>, F.P. Wilson, H. Yuan, B.T.W. Bocher, and W-T. Liu*, 2020. <b>Ecogenomics-based mass balance model reveals the effects of fermentation conditions on microbial activity</b>. <i>Frontiers in Microbiology</i>. <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.595036/full">10.3389/fmicb.2020.595036</a>.<br>
<p style="margin-top: 0.5em;">Identification of optimal fermentation conditions and active microbial populations by applying an ecogenomics-based mass balance approach to organic waste fermentation cultures.</p>

<li>M.K. Nobu†, T. Narihiro†, <b>R. Mei</b>, Y. Kamagata, P-H. Lee, P.K. Lee, M.J., McInerney, and W-T. Liu*, 2020. <b>Catabolism and interactions of uncultured organisms shaped by eco-thermodynamics in methanogenic bioprocesses</b>. <i>Microbiome</i>. <a href="https://link.springer.com/article/10.1186/s40168-020-00885-y">10.1186/s40168-020-00885-y</a>.<br>
<p style="margin-top: 0.5em;">Discovery of diverse uncultured populations driving H2-sensitive metabolisms that are critical for microorganisms overcome a critical obstacle in methanogenic carbon cycles.</p>

<li>L. Ye*, <b>R. Mei</b>, W-T. Liu, H. Ren, and X. Zhang*, 2020. <b>Machine learning-aided analyses of thousands of draft genomes reveal specific features of activated sludge processes</b>. <i>Microbiome</i>. <a href="https://link.springer.com/article/10.1186/s40168-020-0794-3">10.1186/s40168-020-0794-3</a>.<br>
<p style="margin-top: 0.5em;">Recovery of 2,000+ archaeal and bacterial genomes from full-scale activated sludge processes. Identification of functional features vital for microbes to thrive in activated sludge using a machine learning approach.</p>

<li><b>R. Mei</b>, M.K. Nobu, T. Narihiro, and W-T. Liu*, 2020. <b>Metagenomic and
metatranscriptomic analyses revealed uncultured Bacteroidales populations as
the dominant proteolytic amino acid degraders in anaerobic digesters</b>. <i>Frontiers
in Microbiology</i>. <a href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2020.593006/full">10.3389/fmicb.2020.593006</a>.<br>
<p style="margin-top: 0.5em;">Meta-omics characterization of a proteolytic amino acids degrading <i>Bacteroidetes</i> bacterium that is abundant and prevalent in anaerobic wastewater treatment processes.</p>

<li>T.Y.C. Lam†, <b>R. Mei†</b>, Z. Wu, P.K. Lee, W-T. Liu*, and P-H. Lee*, 2020. <b>Superior resolution characterization of microbial diversity in anaerobic digesters using full-length 16S rRNA gene amplicon sequencing</b>. <i>Water Research</i>, <a href="https://www.sciencedirect.com/science/article/abs/pii/S0043135420303523?via%3Dihub">10.1016/j.watres.2020.115815</a>.<br>
<p style="margin-top: 0.5em;">Full-length 16S rRNA gene amplicon sequencing using PacBio Sequel for an accurate phylogenetic investigation of microbiomes in full-scale anaerobic digesters.</p>

<li><b>R. Mei</b>, M.K. Nobu, and W-T. Liu*, 2020. <b>Identifying anaerobic amino acids degraders through the comparison of short-term and long-term enrichments</b>. <i>Environmental Microbiology Reports</i>. <a href="https://ami-journals.onlinelibrary.wiley.com/doi/abs/10.1111/1758-2229.12821">10.1111/1758-2229.12821</a>.<br>
<p style="margin-top: 0.5em;">Continuous enrichment targeting amino acids-degrading populations in methanogenic environments. A clear community shift was observed and described using a culturability index.</p>

<li>H. Sun, <b>R. Mei</b>, X. Zhang, H. Ren, W-T. Liu, and L. Ye*, 2020. <b>Bacterial enrichment in highly-selective acetate-fed bioreactors and its application in rapid biofilm formation</b>. <i>Water Research</i>. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0043135419311339">10.1016/j.watres.2019.115359</a>.<br>
<p style="margin-top: 0.5em;">Identification of novel <i>Acinetobacter</i> species that contribute to sludge granulation and biofilm formation process in highly-selective bioreactors.</p>

<li><b>R. Mei</b> and W-T. Liu*, 2019. <b>Quantifying the contribution of microbial immigration in engineered water systems</b>. <i>Microbiome</i>. <a href="https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0760-0">10.1186/s40168-019-0760-0</a>.<br>
<p style="margin-top: 0.5em;">A perspective on why evaluating the impact of microbial immigration on diversity and function within ecosystems is challenging. An ecogenomics-based mass balance approach may offer a solution.</p>

<li><b>R. Mei</b>, J. Kim, F.P. Wilson, B.T.W. Bocher, and W-T. Liu*, 2019. <b>Coupling growth kinetics model and machine learning quantifies microbial immigration impacts and identifies key environmental parameters in a biological wastewater treatment process</b>. <i>Microbiome</i>. <a href="https://link.springer.com/article/10.1186/s40168-019-0682-x">10.1186/s40168-019-0682-x</a>.<br>
<p style="margin-top: 0.5em;">Quantifying microbial immigration may improve the accuracy of a supervised learning regressor in predicting system performance based on community composition.</p>

<li>H. Yuan, <b>R. Mei</b>, J. Liao, and W-T. Liu*, 2019. <b>Nexus of stochastic and
deterministic processes on microbial community assembly in biological systems</b>. <i>Frontiers in Microbiology</i>. <a href="https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2019.01536/full">10.3389/fmicb.2019.01536</a>.<br>
<p style="margin-top: 0.5em;">Study the core microbial populations and the shift in community composition in lab-scale activated sludge reactors.</p>

<li>A.M. Zealand, <b>R. Mei</b>, P. Papachristodoulou, A. P. Roskilly, W-T. Liu, and D.W. Graham*, 2019. <b>Molecular microbial ecology of stable versus failing rice
straw anaerobic digesters</b>. <i>Microbial Biotechnology</i>. <a href="https://ami-journals.onlinelibrary.wiley.com/doi/full/10.1111/1751-7915.13438">10.1111/1751-7915.13438</a>.<br>
<p style="margin-top: 0.5em;">Assessment of how feeding frequency and organic loading rate impacts microbial communities and biogas production in the treatment of waste rice straw.</p>

<li><b>R. Mei</b>, M.K. Nobu, T. Narihiro, J. Yu, A. Sathyagal, E. Willman, and W-T. Liu*, 2018. <b>Novel <i>Geobacter</i> species and diverse methanogens contribute to enhanced methane production in media-added methanogenic reactors</b>. <i>Water Research</i>. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0043135418308236">10.1016/j.watres.2018.10.026</a>.<br>
<p style="margin-top: 0.5em;">Identification of a phylogenetically novel <i>Geobacter</i> species that could lead to enhanced methane production in softdrink wastewater treatment.</p>

<li>T. Narihiro†, M.K. Nobu†, B.T.W. Bocher, <b>R. Mei</b>, and W-T. Liu*, 2018. <b>Co‐occurrence network analysis reveals thermodynamics‐driven microbial
interactions in methanogenic bioreactors</b>. <i>Environmental Microbiology
Reports</i>. <a href="https://ami-journals.onlinelibrary.wiley.com/doi/abs/10.1111/1758-2229.12689">10.1111/1758-2229.12689</a>.<br>
<p style="margin-top: 0.5em;">Characterization of the interaction of microbial populations in petrochemical wastewater treatment bioreactors, focusing on organisms from multiple niches orchestrating in metabolic networks.</p>

<li>A.M. Zealand, <b>R. Mei</b>, P. Papachristodoulou, A. P. Roskilly, W-T. Liu, and D.W. Graham*, 2018. <b>Microbial community composition and diversity in rice
straw digestion bioreactors with and without dairy manure</b>. <i>Applied
Microbiology and Biotechnology</i>. <a href="https://link.springer.com/article/10.1007/s00253-018-9243-7">10.1007/s00253-018-9243-7</a>.<br>
<p style="margin-top: 0.5em;">Assessment of how adding dairy manure impacts microbial communities and biogas production in the treatment of waste rice straw.</p>

<li>M.K. Nobu†, T. Narihiro†, M. Liu, K. Kuroda, <b>R. Mei</b>, and W-T. Liu*, 2017. <b>Thermodynamically diverse syntrophic aromatic compound catabolism</b>. <i>Environmental Microbiology</i>. <a href="https://ami-journals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.13922">10.1111/1462-2920.13922</a>.<br>
<p style="margin-top: 0.5em;">Investigation of the catabolic diversity and flexibility of bacteria that circumvent thermodynamic restrictions to metabolize biochemically recalcitrant and environmentally relavent aromatic compounds.</p>

<li><b>R. Mei</b>, M.K. Nobu, T. Narihiro, K. Kuroda, J. Sierra, Z. Wu, L, Ye, P.K. Lee, P-H. Lee, J.B. van Lier, M.J., McInerney, Y. Kamagata, and W-T. Liu*, 2017. <b>Operation-driven heterogeneity and overlooked feed-associated populations in anaerobic digester microbiome</b>. <i>Water Research</i>. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0043135417306231">10.1016/j.watres.2017.07.050</a>.<br>
<p style="margin-top: 0.5em;">Analysis of global anaerobic digester microbiome, uncovering vast heterogeneity and inferring the cause.</p>

<li>K. Kuroda†, M.K. Nobu†, <b>R. Mei</b>, T. Narihiro, B.T.W. Bocher, T. Yamaguchi, and W-T. Liu*, 2016. <b>A single-granule-level approach reveals ecological heterogeneity in
an upflow anaerobic sludge blanket reactor</b>. <i>PLOS ONE</i>. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0167788">10.1371/journal.pone.0167788</a>.<br>
<p style="margin-top: 0.5em;">Investigation of the heterogeneity of individual granular micro-ecosystems in lab- and full-scale petrochemical wastewater treatment bioreactors.</p>

<li><b>R. Mei</b>, T. Narihiro, M.K. Nobu, and W-T. Liu*, 2016. <b>Effects of heat shocks on microbial community structure and microbial activity of a methanogenic enrichment degrading benzoate</b>. <i>Letters in Applied Microbiology</i>. <a href="https://academic.oup.com/lambio/article-abstract/63/5/356/6699127">10.1111/lam.12629</a>.<br>
<p style="margin-top: 0.5em;">Demonstration of how a microbial consortium resists and recovers from heat shock with varying intensity.</p>

<li>M.K. Nobu, T. Narihiro, K. Kuroda, <b>R. Mei</b>, and W-T. Liu*, 2016. <b>Chasing the
elusive Euryarchaeota class WSA2: genomes reveal a uniquely fastidious
methyl-reducing methanogen</b>. <i>The ISME Journal</i>. <a href="https://www.nature.com/articles/ismej201633">10.1038/ismej.2016.33</a>.<br>
<p style="margin-top: 0.5em;">The first detailed metagenomic characterization of a methanogen class that is prevalent in anoxic ecosystems.</p>

<li><b>R. Mei</b>, T. Narihiro, M.K. Nobu, K. Kuroda, and W-T. Liu*, 2016. <b>Evaluating digestion efficiency in full-scale anaerobic digesters by identifying active microbial populations through the lens of microbial activity</b>. <i>Scientific Reports</i>. <a href="https://www.nature.com/articles/srep34090">10.1038/srep34090</a>.<br>
<p style="margin-top: 0.5em;">High-throughput, quantitative determination of how microbial species behave after migrating from one ecosystem to another, which is associated with the performance of wastewater treatment.</p>

<li>T. Narihiro, N. Kim, <b>R. Mei</b>, M.K. Nobu, and W-T. Liu*, 2015. <b>Microbial
community analysis of anaerobic reactors treating soft drink wastewater</b>. <i>PLOS ONE</i>. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0119131">10.1371/journal.pone.0119131</a>.<br>
<p style="margin-top: 0.5em;">Description of the microbial community in an efficient system treating softdrink wastewater.</p>

<li>X. Pan, S. Geng, X. Lv, <b>R. Mei</b>, J. Jiangyang, Y. Wang, L. Xu, X. Liu, Y. Tang, G. Wang, and X. Wu*, 2015. <b><i>Defluviimonas alba</i> sp. nov., isolated from an oilfield</b>. <i>International Journal of Systematic and Evolutionary Microbiology</i>. <a href="https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.000181">10.1099/ijs.0.000181</a>.<br>
<p style="margin-top: 0.5em;">Description of a novel bacterial species isolated from oilfield.</p>

<li>S. Geng, X. Pan, <b>R. Mei</b>, Y. Wang, X. Liu, X. Wang, Y. Tang, and X. Wu*, 2015. <b><i>Glycocaulis alkaliphilus</i> sp. nov., a dimorphic prosthecate bacterium
isolated from crude oil</b>. <i>International Journal of Systematic and Evolutionary Microbiology</i>. <a href="https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijs.0.000023">10.1099/ijs.0.000023</a>.<br>
<p style="margin-top: 0.5em;">Description of a novel bacterial species isolated from crude oil.</p>

<li>S. Geng, X. Pan, <b>R. Mei</b>, Y. Wang, J. Sun, X. Liu, Y. Tang, and X. Wu*, 2015. <b><i>Paradevosia shaoguanensis</i> gen. nov., sp. nov., isolated from a coking wastewater</b>. <i>Current Microbiology</i>. <a href="https://link.springer.com/article/10.1007/s00284-014-0689-2">10.1007/s00284-014-0689-2</a>.<br>
<p style="margin-top: 0.5em;">Description of a novel bacterial species isolated from coking wastewater.</p>

<li>X. Pan, S. Geng, <b>R. Mei</b>, Y. Wang, H. Cai, X. Liu, Y. Tang, Y. Nie, S. Ye, and X. Wu*, 2014. <b><i>Nitratireductor shengliensis</i> sp. nov., isolated from
an oil-polluted saline soil</b>. <i>Current Microbiology</i>. <a href="https://link.springer.com/article/10.1007/s00284-014-0624-6">10.1007/s00284-014-0624-6</a>.<br>
<p style="margin-top: 0.5em;">Description of a novel bacterial species isolated from oil-polluted saline soil.</p>

<li>S. Geng, X. Pan, <b>R. Mei</b>, Y. Wang, J. Sun, X. Liu, Y. Tang, and X. Wu*, 2014. <b><i>Ottowia shaoguanensis</i> sp. nov., isolated from coking wastewater</b>. <i>Current Microbiology</i>. <a href="https://link.springer.com/article/10.1007/s00284-013-0481-8">10.1007/s00284-013-0481-8</a>.<br>
<p style="margin-top: 0.5em;">Description of a novel bacterial species isolated from coking wastewater.</p>
</ol>

<h2>Book chapter</h2>
<ol reversed align="justify">
<li>T. Narihiro, M.K. Nobu, <b>R. Mei</b>, and W-T. Liu*, 2015. <b>Microbial community involved in anaerobic purified terephthalic acid treatment process</b>. <i>Anaerobic
Biotechnology: Environmental Protection and Resource Recovery</i>, Imperial
College Press, London. <a href="https://www.worldscientific.com/doi/abs/10.1142/9781783267910_0003">10.1142/9781783267910_0003</a>.<br>
<p style="margin-top: 0.5em;">Summary of the microbial ecology involved in the treatment of an important aromatic compound in the petrochemical industry.</p>
</ol>
</body>


