---
layout: splash
classes: wide
author_profile: false
---

<h2 class="w3-padding-medium w3-text-white w3-center">Peer-reviewed journal publications</h2>
<p class="w3-text-white w3-center">*Corresponding author, †Co-first authors.</p>

<ol reversed class="w3-margin-right">
	
	<div class="w3-padding-48 w3-grey">
	<li>Y. Hirakata, <b>R. Mei</b>, K. Morinaga, T. Katayama, H. Tamaki, X. Meng, T. Watari, T. Yamaguchi, M. Hatamoto, and M.K. Nobu*, 2023. <b>Discovery of anaerobic bacterial scavengers of dead cells</b>. <i>The ISME Journal</i>. <a href="https://academic.oup.com/ismej/article/17/12/2279/7506654">10.1038/s41396-023-01538-2</a>.</br>
	Cultivation and characterization of syntrophic Bacteroidota species that scavenge dead cells using distinct strategies and contribute to the cycle of life and death in oxygen-depleted environments.
	</div>
	
	<li><b>R. Mei</b>, M. Kaneko, H. Imachi, and M.K. Nobu*, 2023. <b>The origin and evolution of methanogenesis and Archaea are intertwined</b>. <i>PNAS Nexus</i>. <a href="https://academic.oup.com/pnasnexus/article/2/2/pgad023/7010768">10.1093/pnasnexus/pgad023</a>.</br>   
	New evidence for the antiquity of methanogenesis and its influence on the transition from ancestral archaea to the diversity prominent today.	
	
	<li><b>R. Mei</b> and W-T. Liu*, 2022. <b>Meta-omics supervised characterization of respiration activities associated with microbial immigrants in anaerobic sludge digesters</b>. <i>Environmental Science & Technology</i>. <a href="https://pubs.acs.org/doi/abs/10.1021/acs.est.2c01029">10.1021/acs.est.2c01029</a>.</br>
	Meta-omics-supervised characterization of microbial immigrants in anaerobic sludge digesters and revealed their activities of anaerobic respiration.
	
	<li>J. Kim†, <b>R. Mei†</b>, F.P. Wilson, H. Yuan, B.T.W. Bocher, and W-T. Liu*, 2020. <b>Ecogenomics-based mass balance model reveals the effects of fermentation conditions on microbial activity</b>. <i>Frontiers in Microbiology</i>. <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2020.595036/full">10.3389/fmicb.2020.595036</a>.</br>
	Identification of optimal fermentation conditions and active microbial populations by applying an ecogenomics-based mass balance approach to organic waste fermentation cultures.
	
	<li>M.K. Nobu†, T. Narihiro†, <b>R. Mei</b>, Y. Kamagata, P-H. Lee, P.K. Lee, M.J., McInerney, and W-T. Liu*, 2020. <b>Catabolism and interactions of uncultured organisms shaped by eco-thermodynamics in methanogenic bioprocesses</b>. <i>Microbiome</i>. <a href="https://link.springer.com/article/10.1186/s40168-020-00885-y">10.1186/s40168-020-00885-y</a>.</br>
	Meta-omics analysis of 17 anaerobic digesters and discovery of diverse uncultured populations driving H2-sensitive metabolisms that are critical for microorganisms overcome a critical obstacle in methanogenic carbon cycles.
	
	
	
	
	
	
	
	
	
	
	
	
	<li><b>R. Mei</b>, M.K. Nobu, and W-T. Liu<sup>*</sup>, 2020. <b>Identifying anaerobic amino acids degraders through the comparison of short-term and long-term enrichments</b>. <i>Environmental Microbiology Reports</i>. <a href="https://ami-journals.onlinelibrary.wiley.com/doi/abs/10.1111/1758-2229.12821">10.1111/1758-2229.12821</a>.</br>
	Continuous enrichment targeting amino acids-degrading populations in methanogenic environments. A clear community shift was observed and described using a culturability index. 
	
	
	
	
	
	
	
	
	
	
	
	
         			<li class="medium">Lam YC†, <b>Mei R</b>†, Wu Z, Lee PK, Liu WT<sup>*</sup> and Lee PH<sup>*</sup>, 2020. Superior resolution characterization of microbial diversity in anaerobic digesters using full-length 16S rRNA gene amplicon sequencing. <i>Water Research</i>, 178, 115175.</br></br>
         			
				<li class="medium"><b>Mei R</b> and Liu WT<sup>*</sup>, 2019. Quantifying the contribution of microbial immigration in engineered water systems. <i>Microbiome</i>, 7, 144.</br></br>
				<li class="medium"><b>Mei R</b>, Kim J, Wilson FP, Bocher BTW and Liu WT<sup>*</sup>, 2019. Coupling growth kinetics model and machine learning quantifies microbial immigration impacts and identifies key environmental parameters in a biological wastewater treatment process. <i>Microbiome</i>, 7, 65.</br></br>
				<li class="medium"><b>Mei R</b>, Nobu MK, Narihiro T, Yu J, Sathyagal A, Willman E and Liu WT<sup>*</sup>, 2018. Novel Geobacter species and diverse methanogens contribute to enhanced methane production in media-added methanogenic reactors. <i>Water Research</i>, 147, pp.403-412.</br></br>
				<li class="medium"><b>Mei R</b>, Nobu MK, Narihiro T, Kuroda K, Sierra J, Wu ZY, Ye L, Lee PKH, Lee PH, van Lier JB, McInerney MJ, Kamagata Y and Liu WT<sup>*</sup>, 2017. Operation-driven heterogeneity and overlooked feed-associated populations in anaerobic digester microbiome. <i>Water Research</i>, 124, pp.77-84.</br></br>
				<li class="medium"><b>Mei R</b>, Narihiro T, Nobu MK, Kuroda K and Liu WT<sup>*</sup>, 2016. Evaluating digestion efficiency in full-scale anaerobic digesters by identifying active microbial populations through the lens of microbial activity. <i>Scientific Reports</i>, 6:34090.</br></br>
				<li class="medium"><b>Mei R</b>, Narihiro T, Nobu MK and Liu WT<sup>*</sup>, 2016. Effects of heat shocks on microbial community structure and microbial activity of a methanogenic enrichment degrading benzoate. <i>Letters in Applied Microbiology</i>, 63(5), pp.356-362.</br></br>
</ol>		
											
1.	Hirakata, Y., Mei, R., Morinaga K., Katayama T., Tamaki H., Meng X.Y., Watari T., Yamaguchi T., Hatamoto M., & Nobu, M.K.*, 2023. Discovery of anaerobic bacterial scavengers of dead cells. The ISME Journal, 17 (12). 10.1038/s41396-023-01538-2
2.	Mei, R., Kaneko, M., Imachi, H., & Nobu, M.K.*, 2023. The origin and evolution of methanogenesis and Archaea are intertwined. PNAS Nexus, 2 (2). 10.1093/pnasnexus/pgad023
3.	Mei, R. & Liu, W.T.*, 2022. Meta-omics supervised characterization of respiration activities associated with microbial immigration in anaerobic digestion. Environmental Science & Technology, 56 (10). 10.1021/acs.est.2c01029
4.	
5.	
6.	Ye, L.*, Mei, R., Liu, W.T., Ren H., & Zhang, X.X.*, 2020. Machine learning-aided analyses of thousands of draft genomes reveal specific features of activated sludge processes. Microbiome, 8. 10.1186/s40168-020-0794-3
7.	Mei, R., Nobu, M.K., Narihiro, T., & Liu, W.T.*, 2020. Metagenomic and metatranscriptomic analyses revealed uncultured Bacteroidales populations as the dominant proteolytic amino acid degraders in anaerobic digesters. Frontiers in Microbiology, 11. 10.3389/fmicb.2020.593006
8.	Lam, Y.C.†, Mei, R.†, Wu, Z., Lee, P.K., Liu, W.T.*, & Lee, P.H.*, 2020. Superior resolution characterization of microbial diversity in anaerobic digesters using full-length 16S rRNA gene amplicon sequencing. Water Research, 178. 10.1016/j.watres.2020.115815
9.	Mei, R., Nobu, M.K., & Liu, W.T.*, 2020. Identifying anaerobic amino acids degraders through the comparison of short-term and long-term enrichments. Environmental Microbiology Reports, 12 (2). 10.1111/1758-2229.12821
10.	Sun, H., Mei, R., Zhang, X.X., Ren H., Liu, W.T., & Ye, L.*, 2019. Bacterial enrichment in highly selective acetate-fed bioreactors and its application in rapid biofilm formation. Water Research, 170. 10.1016/j.watres.2019.115359
11.	Mei, R. & Liu, W.T.*, 2019. Quantifying the contribution of microbial immigration in engineered water systems. Microbiome, 7. 10.1186/s40168-019-0760-0
12.	Mei, R., Kim, J., Wilson, F.P., Bocher, B.T.W., & Liu, W.T.*, 2019. Coupling growth kinetics model and machine learning quantifies microbial immigration impacts and identifies key environmental parameters in a biological wastewater treatment process. Microbiome, 7. 10.1186/s40168-019-0682-x
13.	Yuan, H., Mei, R., Liao, J., & Liu, W.T.*, 2019. Nexus of stochastic and deterministic processes on microbial community assembly in biological systems. Frontiers in Microbiology, 10. 10.3389/fmicb.2019.01536
14.	Zealand, A. M., Mei, R., Papachristodoulou, P., Roskilly, A. P., Liu, W. T., & Graham, D. W.*, 2019. Molecular microbial ecology of stable versus failing rice straw anaerobic digesters. Microbial Biotechnology, 12 (5). 10.1111/1751-7915.13438
15.	Mei, R., Nobu, M.K., Narihiro, T., Yu, J., Sathyagal, A., Willman, E., & Liu, W.T.*, 2018. Novel Geobacter species and diverse methanogens contribute to enhanced methane production in media-added methanogenic reactors. Water Research, 147. 10.1016/j.watres.2018.10.026
16.	Narihiro, T.†, Nobu, M.K.†, Bocher, B.T.W., Mei, R., & Liu, W.T.*, 2018. Co‐occurrence network analysis reveals thermodynamics‐driven microbial interactions in methanogenic bioreactors. Environmental Microbiology Reports, 10 (6). 10.1111/1758-2229.12689
17.	Zealand, A. M., Mei, R., Papachristodoulou, P., Roskilly, A. P., Liu, W. T., & Graham, D. W.*, 2018. Microbial community composition and diversity in rice straw digestion bioreactors with and without dairy manure. Applied Microbiology and Biotechnology, 102 (19). 10.1007/s00253-018-9243-7
18.	Nobu, M.K.†, Narihiro, T.†, Liu, M.M., Kuroda, K., Mei, R., & Liu, W.T.*, 2017. Thermodynamically diverse syntrophic aromatic compound catabolism. Environmental Microbiology, 19 (11). 10.1111/1462-2920.13922
19.	Mei, R., Nobu, M.K., Narihiro, T., Kuroda, K., Sierra, J., Wu, Z.Y., Ye, L., Lee, P.K.H., Lee, P.H., van Lier,. J.B., McInerney, M.J., Kamagata, Y., & Liu, W.T.*, 2017. Operation-driven heterogeneity and overlooked feed-associated populations in anaerobic digester microbiome. Water Research, 124. 10.1016/j.watres.2017.07.050
20.	Kuroda, K.†, Nobu, M.K.†, Mei, R., Narihiro, T., Bocher, B.T., Yamaguchi, T., & Liu, W.T.*, 2016. A single-granule-level approach reveals ecological heterogeneity in an upflow anaerobic sludge blanket reactor. PLOS ONE, 11 (12). 10.1371/journal.pone.0167788
21.	Mei, R., Narihiro, T., Nobu, M.K., & Liu, W.T.*, 2016. Effects of heat shocks on microbial community structure and microbial activity of a methanogenic enrichment degrading benzoate. Letters in Applied Microbiology, 63 (5). 10.1111/lam.12629
22.	Nobu, M.K., Narihiro, T., Kuroda, K., Mei, R., & Liu, W.T.*, 2016. Chasing the elusive Euryarchaeota class WSA2: genomes reveal a uniquely fastidious methyl-reducing methanogen. The ISME Journal, 10 (10). 10.1038/ismej.2016.33
23.	Mei, R., Narihiro, T., Nobu, M.K., Kuroda, K., & Liu, W.T.*, 2016. Evaluating digestion efficiency in full-scale anaerobic digesters by identifying active microbial populations through the lens of microbial activity. Scientific Reports, 6. 10.1038/srep34090
24.	Pan, X.C.†, Geng, S., Lv, X.L.†, Mei, R., Jiangyang, J.H., Wang, Y.N., Xu, L., Liu, X.Y., Tang, Y.Q., Wang, G.J., & Wu, X.L.*, 2015. Defluviimonas alba sp. nov., isolated from an oilfield. International Journal of Systematic and Evolutionary Microbiology, 65 (6). 10.1099/ijs.0.000181
25.	Narihiro, T., Kim, N.K., Mei, R., Nobu, M.K., & Liu, W.T.*, 2015. Microbial community analysis of anaerobic reactors treating soft drink wastewater. PLOS ONE, 10 (3). 10.1371/journal.pone.0119131
26.	Geng, S., Pan, X.C., Mei, R., Wang, Y.N., Liu, X.Y., Wang, X.B., Tang, Y.Q., & Wu, X.L.*, 2015. Glycocaulis alkaliphilus sp. nov., a dimorphic prosthecate bacterium isolated from crude oil. International Journal of Systematic and Evolutionary Microbiology, 65 (3). 10.1099/ijs.0.000023
27.	Geng, S., Pan, X.C., Mei, R., Wang, Y.N., Sun, J.Q., Liu, X.Y., Tang, Y.Q., & Wu, X.L.*, 2015. Paradevosia shaoguanensis gen. nov., sp. nov., isolated from a coking wastewater. Current Microbiology, 70 (1). 10.1007/s00284-014-0689-2
28.	Pan, X.C., Geng, S., Mei, R., Wang, Y.N., Cai, H., Liu, X.Y., Tang, Y.Q., Nie, Y., Ye, S.Y., & Wu, X.L.*, 2014. Nitratireductor shengliensis sp. nov., isolated from an oil-polluted saline soil. Current Microbiology, 69 (4). 10.1007/s00284-014-0624-6
29.	Geng, S., Pan, X.C., Mei, R., Wang, Y.N., Sun, J.Q., Liu, X.Y., Tang, Y.Q., & Wu, X.L.*, 2014. Ottowia shaoguanensis sp. nov., isolated from coking wastewater. Current Microbiology, 68 (3). 10.1007/s00284-013-0481-8



